### ISSUE TITLE
Participaremos!! - en español.

### ISSUE THREAD

**Issue by alegalviz** [NaNoGenMo/2016#76]
I and my coworkers will participate generating a Novel in Spanish.

Ideas:
- Change words by synonyms in some book
- Reagroup tweets of someone by chapters
- Merge some books and stories like nordic stories with marvel, with saint seya, etc.

**Comment by hugovk** [comment]
Non-English entries are very welcome, and also team efforts! And I think this may be the first team entry. Good luck!
